src/page/index.ts

```diff
@@ -0,0 +1,9 @@
+export { MenuContentPage } from './menu-content.page';
+export { ProductListPage } from './product-list.page';
+export { ProductAddedModalPage } from './product-added-modal.page';
+export { OrderSummaryPage } from './order-summary.page';
+export { SignInStepPage } from './sign-in-step.page';
+export { AddresStepPage } from './address-step.page';
+export { ShippingStepPage } from './shipping-step.page';
+export { PaymentStepPage } from './payment-step.page';
+export { BankPaymentPage } from './bank-payment.page';
```

Comment:
You are missing `summary-step.page.ts` and confusing that page with the `order-summary.page.ts`, which should contain the last step of the buying process.

Reply:
Yes, I missed it, thanks for the advice.
I think it is fixed now :)